bundles/org.openhab.binding.sony/README-IRCC.md

```diff
@@ -0,0 +1,209 @@
+# IRCC
+
+IRCC (otherwise know as IRCC-IP - InfraRed Compatible Control over Internet Protocol) will allow you to send IR commands to the device over IP.
+
+Many Sony products (TVs, AV systems, disc players) provided an IRCC service that provides minimal control to the device and some minimal feedback (via polling) depending on the version.  From my research, their appears to be 5 versions of this service:
```

Comment:
Please put single sentences on a new line to make reviewing (later) changes easier.